lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer_test.go

```diff
@@ -38,12 +38,50 @@ import (
 	"magma/orc8r/cloud/go/services/configurator"
 	configurator_storage "magma/orc8r/cloud/go/services/configurator/storage"
 	configurator_test_init "magma/orc8r/cloud/go/services/configurator/test_init"
+	orc8rModels "magma/orc8r/cloud/go/services/orchestrator/obsidian/models"
 	"magma/orc8r/cloud/go/sqorc"
 	"magma/orc8r/cloud/go/storage"
 	"magma/orc8r/cloud/go/syncstore"
 	"magma/orc8r/lib/go/protos"
 )
 
+func TestListSuciProfiles(t *testing.T) {
+	lte_test_init.StartTestService(t)
+	configurator_test_init.StartTestService(t)
+	storeReader, _ := initializeStore(t)
```

Comment:
Handle err

Reply:
initializeStore(t) doesn't return an error. I've handled the error while creating the network and also while adding test case values.